.travis.yml

```
@@ -1,10 +1,8 @@
 language: go
-env:
-  - GO111MODULE=on
 script: go test -race -cpu 1,2,4 -v -timeout 5m ./...
 sudo: false
 go:
-  - 1.13.x
+  - 1.16.x
```

We should follow Go's guidelines and support the last two major versions:

```suggestion
  - 1.15.x
  - 1.16.x
```

Or we should make the repo private 😉